src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -51,8 +51,12 @@
     private static final Map<String, List<Integer>> CHECK_TO_ANCESTOR_NODES_MAP = new HashMap<>();
 
     static {
+        CHECK_TO_ANCESTOR_NODES_MAP.put("ArrayTrailingComma",
+                Arrays.asList(TokenTypes.ARRAY_INIT));
```

Comment:
ArrayTrailingComma looks good. Please move it to a separate pull request, so we can merge it to master.

Reply:
@baratali ok, separate pr is here: https://github.com/checkstyle/patch-filters/pull/324

Reply:
merged